android/android_env.sh

```diff
@@ -11,32 +22,28 @@ fi
 case $1 in
  armeabi-v7a )
   export CHOST=armv7a-linux-androideabi
-  export NOAPI_PREFIX=${TOOLCHAIN}arm-linux-androideabi
-  export ARCH_FLAGS="-march=armv7-a -mfloat-abi=softfp -mfpu=vfpv3-d16 -O2"
+  export ARCH_FLAGS="-march=armv7-a -mfloat-abi=softfp -mfpu=vfpv3-d16 -O3"
   export ARCH=armeabi-v7a
   export CONFIGURE_PLATFORM="android-arm"
   ARCH_OPTS="--enable-sp-asm"
   ;;
  arm64-v8a )
   export CHOST=aarch64-linux-android
-  export NOAPI_PREFIX=${TOOLCHAIN}aarch64-linux-android
-  export ARCH_FLAGS="-march=armv8-a+crypto -O2"
+  export ARCH_FLAGS="-march=armv8-a+crypto -O3"
   export ARCH=arm64-v8a
   export CONFIGURE_PLATFORM="android-arm64"
-  ARCH_OPTS="--enable-armasm"
+  ARCH_OPTS="--enable-armasm --enable-sp-asm"
```

Comment:
Nice catch. Since this actually fixed a bug, can you make it a separate PR instead of mixing in the CI change PR?